cli/tests/testdata/jupyter/format_test.ts

```diff
@@ -0,0 +1,57 @@
+import vl from "npm:vega-lite-api";
+
+const { format } = Deno.jupyter;
```

Comment:
Wow, this is super surprising - access to `Deno.jupyter` should throw in non-notebook contexts.